index.html

```diff
@@ -271,13 +273,48 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='pt'>
```

Comment:
Both "a AMP" (“a tecnologia AMP”) or "o AMP" (“o projeto AMP”) apply, but only one should be chosen for consistency